specification/web/resource-manager/Microsoft.Web/stable/2023-12-01/WebApps.json

```diff
@@ -28335,6 +28767,97 @@
         }
       }
     },
+    "SiteContainer": {
+      "description": "Container of a site",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "SiteContainer resource specific properties",
+          "type": "object",
+          "properties": {
+            "image": {
+              "description": "Image Name",
+              "type": "string"
+            },
+            "targetPort": {
+              "description": "Target Port",
+              "type": "string"
+            },
+            "isMain": {
+              "description": "IsMain \nSpecifies the entrypoint container of\nthe application",
```

Comment:
could you update the description to let users know what does `true` mean and what does `false` mean?

Reply:
updated.